rules/linux/auditd/lnx_auditd_move_clone_open_tree_mount_partitions.yml

```diff
@@ -0,0 +1,40 @@
+title: Unusual Mount Tree Activity
+id: 6ee3ea09-f9da-406a-9339-35c4b919658a
+status: experimental
+description: |
+    By invoking the move_mount and open_tree syscalls, I was able to clone and mount the /proc filesystem onto the /tmp directory.
+    The operation works by first using open_tree to open the /proc directory tree and then utilizing move_mount to move or mount it onto /tmp/proc.
+    This action is typically used to hide system information, manipulate /proc data, or evade detection by monitoring tools.
+references:
+    - https://lore.kernel.org/all/153313705821.13253.15519769348603350410.stgit@warthog.procyon.org.uk/
+author: Milad Cheraghi
+date: 2025-04-30
+tags:
+    - attack.t1564.005
+    - attack.defense-evasion
+logsource:
+    product: linux
+    service: auditd
+detection:
+    selection_open_tree:
+        syscall: 428
+    selection_move_mount:
+        syscall: 429
```

Comment:
The syscall can not be 428 and 429 in the same time `AND (selection_open_tree AND selection_move_mount)`